docs/spfx/build-for-teams-configure-in-teams.md

```diff
@@ -0,0 +1,69 @@
+
+When building tabs using SharePoint Framework, the generated tab uses the [web part property pane](web-parts/guidance/integrate-web-part-properties-with-sharepoint.md) to let users configure the tab. This saves you a lot of effort. Not only you don’t need to build and maintain a separate configuration UI but you also don’t need to implement any code responsible for storing and managing the settings. All of that is handled automatically for you by SharePoint Framework.
+
+## Configure Microsoft Teams personal apps built using SharePoint Framework
+
+Microsoft Teams personal apps don’t offer any infrastructure for implementing configuration. Instead, following the pattern recommended by Microsoft Teams, personal app’s [settings should be exposed in a separate tab](https://docs.microsoft.com/en-us/microsoftteams/platform/concepts/design/personal-apps).
+
+:::image type="content" source="../images/build-for-teams/build-for-teams-personal-app.png" alt-text="Personal app with multiple tabs including settings built using SharePoint Framework":::
+
+Translating this to your SharePoint Framework solution, it means building a separate web part that contains the configuration UI and which will be used only in the context of the personal app, defining a storage for user’s configuration and extending the personal app definition to contain multiple tabs.
+
+### Personal app configuration UI web part
+
+Each tab in a personal app is mapped to a URL. The easiest way to build UI for configuring your SharePoint Framework web part exposed as a personal app, is by building a separate web part with the configuration UI. This will allow you to optimize the UI for use in the context of the personal app and keep this code separate from your web part.
+
+Since the personal app configuration UI web part is not meant to be used outside of the personal app, in its manifest, you should set the **supportedHosts** property to an empty array.
+
+:::image type="content" source="../images/build-for-teams/build-for-teams-manifest-setttings-webpart.png" alt-text="The supportedHosts property of a SharePoint Framework web part used as a personal app configuration UI":::
+
+The reason you want the **supportedHosts** property to be empty is to prevent the web part from being used in SharePoint but also not include it in the autogenerated Teams manifest. By default, each web part that contains the **TeamsTab** or **TeamsPersonalApp** value in the manifest is included in the generated Teams manifest as a separate Microsoft Teams app. In this case however, you want the personal app to consist of multiple tab, each pointing to a different web part. This can be done only by manually updating the manifest yourself.
```

Comment:
Bold items here should use inline code vs. bold formatting. See previous comment.